### ISSUE TITLE
Wallets update

### ISSUE THREAD

**Issue by Sami32** [thebitcoinhole/software-wallets#383]
10101 is winding down:
https://10101.finance/blog/

Vultisig has been omitted:
https://github.com/Vultisig/

Great work, thank you +1

**Comment by maxirosson** [comment]
Thanks for the comments. I removed 10101. 
Right now I am trying to include the more popular software wallets.There are hundreds and is impossible to keep all of them updated

**Comment by maxirosson** [comment]
Yes, I appreciate the suggestions. I use to include all the suggestions on my backlog and add them when they are popular enough. I would love to include all the available wallets, even the bad ones, so the website can help people to don't install them. But doing that is a big effort right now.

**Comment by Sami32** [comment]
Including all seem an unreachable target for a guy alone, or maybe at the price of having no life.

Perhaps using AI to monitor the products blog and Twitter account to parse in real time the meaningful changes could greatly alleviate your work?

**Comment by maxirosson** [comment]
I have github actions to update software versions and warn me about new release notes. Also some price checkers for some hardware wallets. But the idea in the future is that people can collaborate on the open source projects to keep them updated
